terraform/environment/region/modules/event_bus/main.tf

```diff
@@ -0,0 +1,53 @@
+# Event bus for reduced fees
+
+resource "aws_cloudwatch_event_bus" "reduced_fees" {
```

Comment:
it's probably worth making "reduced fees" more generic everywhere since it is being used for anything now